shell/platform/darwin/ios/rendering_api_selection.mm

```diff
@@ -21,26 +21,34 @@ bool ShouldUseMetalRenderer() {
   // past iOS 10.0. The processor was selected as it is the first version at which Metal was
   // supported. The iOS version floor was selected due to the availability of features used by Skia.
   bool ios_version_supports_metal = false;
+#if TARGET_IPHONE_SIMULATOR
+  if (@available(iOS 13.0, *)) {
```

Comment:
It may be time to put 13.0 in a 10.0 and a `constexpr double` declaration in the header. Maybe something like `constexpr double kMetalOSVersionBaseline = <>` and use that in the @available macros elsewhere. Think of it this way, if we wanted to update the baseline, would modifying just `rendering_api_selection` (header or implementation) be enough? Right now, we would need to modify this translation unit and also hunt for all the places we hardcoded 13.0 or 10.0. Let's isolate them all into one variable that everyone else can refer to.

Reply:
sounds like a plan
Where should I put that constant?

Reply:
Using a `constexpr` is not possible since the version number is not actually a double (it can for instance be 13.0.1) so I used a `#define` instead